samples/spring-boot-auto-config/.cache/formatter-maven-cache.properties

```diff
@@ -0,0 +1,5 @@
+#Fri Jan 21 13:29:12 CET 2022
```

Comment:
Don't think these cache files should be getting commit.

Reply:
removed, thx!